tests/scripts/thread-cert/Cert_5_3_06_RouterIdMask.py

```diff
@@ -100,22 +100,24 @@ def test(self):
 
         # 3 & 4
         # Verify the cost from DUT_LEADER to ROUTER2 goes to infinity in 12 mins.
-        flag_in_id_set = False
         routing_cost = 1
         for i in range(0, 24):
             time.sleep(30)
+            print("sleep-------------------------: %s" %((i + 1) * 30))
```

Comment:
`sleep` here might be a little confusing. It should be fine to only output the timing.